problem_builder/questionnaire.py

```diff
@@ -98,6 +99,9 @@ def parse_xml(cls, node, runtime, keys, id_generator):
 
         # Load XBlock properties from the XML attributes:
         for name, value in node.items():
+            if name not in block.fields:
```

Comment:
@bradenmacdonald heads up - `parse_xml` is going to be removed: https://github.com/open-craft/xblock-mentoring/pull/16